content/api/javascript/quickstart.md

```diff
@@ -24,19 +24,26 @@ It can't:
 
 [Appbase.io - the database service](https://appbase.io) is opinionated about cluster setup and hence doesn't support the Elasticsearch devops APIs. See [rest.appbase.io](https://rest.appbase.io) for a full reference on the supported APIs.
 
-This is a quick start guide to whet the appetite with the possibilities of data streams.
+## Create Cluster OR Use Existing Elasticsearch
 
-## Creating an App
+You can run dedicated Elasticsearch clusters with Appbase.io services to meet your search requirements using [Appbase.io clusters](/docs/hosting/clusters/).
```

Comment:
@lakhansamani Refer to the comment in the Rest quickstart for the exact text to use here.